internal/pkg/term/selector/selector.go

```diff
@@ -221,6 +222,24 @@ func (s *WorkspaceSelect) Service(prompt, help string) (string, error) {
 	return selectedServiceName, nil
 }
 
+// Job fetches all services in the workspace and then prompts the user to select one.
+func (s *WorkspaceSelect) Job(prompt, help string) (string, error) {
+	jobNames, err := s.retrieveWorkspaceJobs()
+	if err != nil {
+		return "", fmt.Errorf("list jobs: %w", err)
+	}
+	if len(jobNames) == 1 {
+		log.Infof("Only found one service in workspace, defaulting to: %s\n", color.HighlightUserInput(jobNames[0]))
+		return jobNames[0], nil
+	}
+
+	selectedJobName, err := s.prompt.SelectOne(prompt, help, jobNames)
```

Comment:
Can we add `prompt.WithFinalMessage("Job name")`?

Reply:
Yes. Should I add the equivalent for `svc_deploy` as well? Currently, we have it for both `svc_init` and `job_init`

Reply:
that'd be awesome!

Reply:
Okay, finally done!